docs/src/man/querying_frameworks.md

```diff
@@ -1,5 +1,101 @@
 # Querying frameworks
```

Comment:
I wonder whether another name wouldn't be more evocative. dplyr says "data manipulation framework". What do you think? It would also be useful to have a few words saying that DataFrames support two such frameworks, which are similar to dplyr and LINQ.

Reply:
I have tried to write up something better :). Commit is pushed.